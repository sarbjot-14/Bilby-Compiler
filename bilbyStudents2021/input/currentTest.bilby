main{


	imm i := [#g,#a,14];
	print i $n;
	
	
}