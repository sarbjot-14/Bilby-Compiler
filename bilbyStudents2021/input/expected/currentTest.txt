[1, 2, 3, 4]
[testing, the, compiler]
